app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java

```diff
@@ -129,33 +148,173 @@ public void onHeaderClick(View view) {
 //        startActivity(goHome);
     }
 
-    private void setUpMenu() {
-        mHomeView.addDrawerListener(
-            new DrawerLayout.DrawerListener() {
-                @Override
-                public void onDrawerSlide(View drawerView, float slideOffset) {
-                    if (slideOffset < 0.4) { mSearchView.setLeftMenuOpen(false); }
+    private void setUpRouteSwitcher() {
+        mRouteIndicator = this.findViewById(R.id.route_indicator);
```

Comment:
please leave a comment for what the mRouteIndicator is, also one for mRouteSwitchButton

Reply:
Thought the variable names were descriptive enough, but I've since added comments for the more ambiguous ones.